src/__internal__/input/input-presentation.component.tsx

```diff
@@ -30,8 +30,8 @@ export interface CommonInputPresentationProps extends ValidationProps {
   size?: Sizes;
   /** If true, the component has an icon rendered inside */
   hasIcon?: boolean;
-  /** Specify a custom border radius. Any valid border-radius design token name. */
-  borderRadius?: BorderRadiusType;
+  /** Specify a custom border radius. Any valid string or array of border-radius design tokens. */
```

Comment:
**suggestion** (non-blocking): I'm not sure I like the "any valid string" phrasing when it can only be a design token. It's not actually wrong, since I'm sure you'd argue that in this context "valid" means "equal to a design token name" 😆 But I'd personally prefer to phrase this something like: `any valid border-radius design token, or an array of such tokens`.